src/plugins/timelist/inspector/FilteringComponent.vue

```diff
@@ -24,20 +24,43 @@
     <div v-if="canEdit" class="c-inspect-properties__hint span-all">
       Filter this view by comma-separated keywords.
     </div>
-    <div class="c-inspect-properties__label" title="Filter by keyword.">Filters</div>
-    <div v-if="canEdit" class="c-inspect-properties__value" :class="{ 'form-error': hasError }">
+    <div class="c-inspect-properties__label" title="Filter by keyword.">Name Filters</div>
```

Comment:
titles should also have aria-label's which match

Reply:
Done.